src/Members/PlatformTrait.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the fusonic/webapp package.
+ *
+ * (c) Fusonic GmbH <office@fusonic.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Fusonic\WebApp\Members;
+
+/**
+ * Adds support for the `platform` member.
+ *
+ * @package Fusonic\WebApp
+ *
+ * @see PlatformInterface
+ * @see https://www.w3.org/TR/appmanifest/#platform-member
+ */
+trait PlatformTrait
+{
+    private $platform;
+
+    /**
+     * Returns the intended target platform for this object.
+     *
+     * @return  string|null
+     */
+    public function getPlatform()
+    {
+        return $this->platform;
+    }
+
+    /**
+     * Sets the intended target platform for this object.
+     *
+     * @param   string              $platform
+     *
+     * @return  $this
+     */
+    public function setPlatform($platform)
+    {
+        $this->platform = $platform;
+        return $this;
+    }
+}
```

Comment:
I think it's the way to go, because `platform` is specified as a "multi-purpose member" and the trait is tested by its own unit test – it wouldn't be wise to test the same functionality in multiple objects.